src-docs/src/views/selectable/selectable_example.js

```diff
@@ -225,10 +225,12 @@ export const SelectableExample = {
           <h3>Width and height</h3>
           <p>
             The width has been made to always be 100% of its container,
-            including stretching the search input. By default, the height is
-            capped at showing up to 7.5 items. It shows half of the last one to
-            help indicate that there are more options to scroll to. To stretch
-            the box to fill its container, pass &apos;full&apos; to the{' '}
+            including stretching the search input. When used inside of
+            `EuiPopover`, we recommend setting a width (or min/max values) to
```

Comment:
```suggestion
            <Link to="fill_in"><strong>EuiPopover</strong></Link>, we recommend setting a width (or min/max values) to
```

Though this sentence makes me think I can pass `width` directly to EuiPopover or EuiFilterGroup. Can you be more explicit?